include/bluetooth/gatt.h

```diff
@@ -625,7 +625,11 @@ ssize_t bt_gatt_attr_read_chrc(struct bt_conn *conn,
 #if IS_ENABLED(CONFIG_BT_SETTINGS_CCC_LAZY_LOADING)
 	#define BT_GATT_CCC_MAX (CONFIG_BT_MAX_CONN)
 #else
-	#define BT_GATT_CCC_MAX (CONFIG_BT_MAX_PAIRED + CONFIG_BT_MAX_CONN)
+	#if defined(CONFIG_BT_CONN)
+		#define BT_GATT_CCC_MAX (CONFIG_BT_MAX_PAIRED + CONFIG_BT_MAX_CONN)
+	#else
+		#define BT_GATT_CCC_MAX 0
+	#endif
 #endif
```

Comment:
Minor: This could be 
```
#if IS_ENABLED(CONFIG_BT_SETTINGS_CCC_LAZY_LOADING)
	#define BT_GATT_CCC_MAX (CONFIG_BT_MAX_CONN)
#elif defined(CONFIG_BT_CONN)
	#define BT_GATT_CCC_MAX (CONFIG_BT_MAX_PAIRED + CONFIG_BT_MAX_CONN)
#else
	#define BT_GATT_CCC_MAX 0
#endif

Reply:
Ok, changed. Also switched from IS_ENABLED to defined (for consistency).